SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -49,6 +49,7 @@ USpatialNetDriver::USpatialNetDriver(const FObjectInitializer& ObjectInitializer
 	, bWaitingForAcceptingPlayersToSpawn(false)
 	, NextRPCIndex(0)
 	, TimeWhenPositionLastUpdated(0.f)
+	, CreateServerWorkerEntityNumberOfAttempts(0)
```

Comment:
Can we add the retry count as an arg to the delegate, this means we don't need to have a new netdriver member?